lib/livebook/notebook/explore/kino/intro_to_kino.livemd

```diff
@@ -96,16 +96,14 @@ side-by-side. We can actually gather information about these
 processes and render it as a table:
 
 ```elixir
-processes = Process.list() |> Enum.map(&Process.info/1)
-```
+keys = [:registered_name, :initial_call, :reductions, :stack_size]
 
-We can pick the data keys that are relevant for us:
+processes =
+  for pid <- Process.list(), info = Process.info(pid) do
+    for key <- keys, into: %{}, do: {key, info[key]}
+  end
 
-```elixir
-Kino.DataTable.new(
-  processes,
-  keys: [:registered_name, :initial_call, :reductions, :stack_size]
-)
+Kino.DataTable.new(processes)
```

Comment:
We don't allow lists of keyword lists anymore? If not, shouldn't we?

I would love if we could still keep the same code as before, perhaps by specifying `:only` instead of `:keys`.